content/en/security/code_security/iast/setup/compatibility/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Compatibility Requirements
```

Comment:
The content on this page should move into a Setup page that lives under the Runtime Code Analysis (IAST) heading, similar to the SAST and SCA structures